crates/viewer/re_space_view_graph/src/error.rs

```diff
@@ -0,0 +1,19 @@
+use re_log_types::EntityPath;
+use re_types::datatypes;
+use re_viewer_context::SpaceViewSystemExecutionError;
+
+#[derive(thiserror::Error, Debug)]
+pub enum Error {
+    #[error("edge has unknown node")]
+    EdgeUnknownNode,
+
+    #[error("missing layout information for node `{1}` in entity `{0}`")]
+    MissingLayoutInformation(EntityPath, datatypes::GraphNode),
+}
+
+impl From<Error> for SpaceViewSystemExecutionError {
+    fn from(val: Error) -> Self {
+        // TODO(grtlr): Ensure that this is the correct error type.
+        Self::DrawDataCreationError(Box::new(val))
```

Comment:
I'm not sure it is the correct type. If i'm not mistaken that variant relates specifically to GPU draw data to be fed to re_renderer, which we're not using in this view.

We repeatedly run into this limitation of `SpaceViewExecutionError`. We probably should add a variants like `SystemError { identifier: String, error: Box<dyn Error> }`, and `ViewClassError { identifier: String, error: Box<dyn Error> }` so you can actually do this.

@Wumpf do you agree?

Reply:
correct, `DrawData` refer's to re_renderer's `DrawData` here
Agreed, it's time to make the error type more scalable by adding view specific subtypes!

Reply:
I will create a separate PR for this!

Reply:
Corresponding PR: https://github.com/rerun-io/rerun/pull/7942